### PR TITLE
libao: update to 1.2.2.

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

File: srcpkgs/libao/template [hunk 1]
```diff
@@ -10,9 +10,9 @@ conf_files="/etc/libao.conf"
 homepage="http://xiph.org/ao/"
```
Comment (maxice8) [inline srcpkgs/libao/template]: homepage should be above distfiles= and the page is now https://

**Comment by Chocimier** [comment]
Is there a tool to check for these?

**Comment by maxice8** [comment]
nope

**Comment by maxice8** [comment]
There isn't at the moment but one can make and contribute to xlint.

**Pull request merged**